src/modules/Auth/components/Registration/Registration.tsx

```diff
@@ -1,7 +1,213 @@
+                      <input
+                        type="text"
+                        className="floating-input "
+                        {...register('country', {
+                          required: 'Country is required.',
+                        })}
+                         placeholder=""
+                      />
+                      <label>Country</label>
+                    </div>
+                    {errors.country && <span className="text-danger">{errors.country.message}</span>}
+
+                    {/* Password */}
+                    <div className="floating-label    rounded-2 password">
+                    <span className="highlight"></span>
+                    
+                      <input
+                        type={showPassword ? 'text' : 'password'}
+                        className="floating-input "
+                        {...register('password', PasswordValidation)}
+                       placeholder=""
+                      />
+                      <label>Password</label>
+                      {/* <span
+                        className="icon-showpass"
+                        onClick={togglePasswordVisibility}
+                        style={{ cursor: 'pointer' }}
+                      >
+                        {showPassword ? <i className="fa-solid fa-eye"></i> : <i className="fa-solid fa-eye-slash"></i>}
+                      </span> */}
+                    </div>
+                    {errors.password && <span className="text-danger">{errors.password.message}</span>}
+                  </div>
+
+                  {/* Second Column */}
+                  <div className="col-md-12 col-lg-6">
+                    {/* Email */}
+                    <div className="floating-label">
+                    <span className="highlight"></span>
+                      <input
+                        type="text"
+                        className="floating-input "
+                        {...register('email', EmailValidation)}
+                       placeholder=""
+                      />
+                      <label>E-mail</label>
+                    </div>
+                    {errors.email && <span className="text-danger">{errors.email.message}</span>}
+
+                    {/* Phone Number */}
+                    <div className="floating-label    rounded-2 phone">
+                    <span className="highlight"></span>
+                      <input
+                        type="text"
+                        className="floating-input "
+                        {...register('phoneNumber', PhoneNumberValidation)}
+                       placeholder=""
+                      />
+                      <label>Phone Number</label>
+                    </div>
+                    {errors.phoneNumber && <span className="text-danger">{errors.phoneNumber.message}</span>}
+
+                    {/* Confirm Password */}
+                    <div className="floating-label    rounded-2 confirm-password">
```

Comment:
We can create a password input and then re-use it in our application.

Reply:
I create password input but i have problem in show password icon 
didn't work , i try to consider solution but i failed 
can you tell me what i can do to fix this ?

Reply:
What do you mean by "didn't work"?

Reply:
i fix show icon thank you